pkg/orchestration/wiring/kubeingress/plugin.go

```diff
@@ -40,7 +40,6 @@ const (
 	clusterHostPath = "k8s.atl-paas.net"
 
 	servicePostfix = "service"
-	ingressPostfix = "ingress"
```

Comment:
With this PR the ingress resource in Bundle and the Ingress object don't have this postfix.